MAUI/Funnel-Charts/Tooltip.md

```diff
@@ -0,0 +1,131 @@
+
+<chart:SfFunnelChart EnableTooltip="True">
+. . .
+</chart:SfFunnelChart>
+
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfFunnelChart chart = new SfFunnelChart();
+. . .      
+chart.EnableTooltip = true;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Tooltip support in MAUI chart](Tooltip_images/MAUI_chart_tooltip.png)
+
+The [ChartTooltipBehavior](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartTooltipBehavior.html) is used to customize the tooltip. For customizing the tooltip, create an instance [ChartTooltipBehavior](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartTooltipBehavior.html) and set it to the [TooltipBehavior](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartBase.html#Syncfusion_Maui_Charts_ChartBase_TooltipBehavior) property of [SfFunnelChart](). The following properties are used to customize the tooltip:
+
+* [Background](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartTooltipBehavior.html#Syncfusion_Maui_Charts_ChartTooltipBehavior_Background) of type `Brush`, indicates background color to the tooltip label.
+* [FontAttributes](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartTooltipBehavior.html#Syncfusion_Maui_Charts_ChartTooltipBehavior_FontAttributes) of type `FontAttributes`, indicates the font style for the label.
+* [FontFamily](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartTooltipBehavior.html#Syncfusion_Maui_Charts_ChartTooltipBehavior_FontFamily) of type `string`, indicates the font family name for the label.
+* [FontSize](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartTooltipBehavior.html#Syncfusion_Maui_Charts_ChartTooltipBehavior_FontSize) of type `float`, indicates the font size for the label.
+* [Duration](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartTooltipBehavior.html#Syncfusion_Maui_Charts_ChartTooltipBehavior_Duration) of type `int`, indicates the duration of the tooltip text in seconds.
+* [Margin](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartTooltipBehavior.html#Syncfusion_Maui_Charts_ChartTooltipBehavior_Margin) of type `Thickness`, indicates the margin of the label.
+* [TextColor](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartTooltipBehavior.html#Syncfusion_Maui_Charts_ChartTooltipBehavior_TextColor) of type `Color`, indicates the text color of the label.
+
+{% tabs %}
+
+{% highlight xml %}
+
+<chart:SfFunnelChart EnableTooltip="True">
+. . .
+<chart:SfFunnelChart.TooltipBehavior>
+    <chart:ChartTooltipBehavior/>
+</chart:SfFunnelChart.TooltipBehavior>
+</chart:SfFunnelChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfFunnelChart chart = new SfFunnelChart();
+. . .
+chart.EnableTooltip = true;
+chart.TooltipBehavior = new ChartTooltipBehavior();
+. . .
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Template
+
+Funnel chart provides support to customize the appearance of the tooltip by using the [TooltipTemplate]() property.
+
```

Comment:
Expected to show case additional information other than the default UI, the TooltipTemplate support helps.